sdks/python/apache_beam/transforms/stats.py

```diff
@@ -430,11 +430,11 @@ class ApproximateQuantilesCombineFn(CombineFn):
   def __init__(
       self, num_quantiles, buffer_size, num_buffers, key=None, reverse=False):
     if key:
-      self._comparator = lambda a, b: (key(a) < key(b)) - (key(a) > key(b)) \
-        if reverse else (key(a) > key(b)) - (key(a) < key(b))
+      self._comparator = lambda a, b: (key(a) < key(b)) ^ (key(a) > key(b)) \
```

Comment:
Maybe I am missing something here, but I think here we use - to get +1, 0 or -1 and ^ operator would not give us.

Reply:
good catch! fixed.